mlflow/server/handlers.py

```diff
@@ -742,7 +742,7 @@ def _log_param():
         schema={
             "run_id": [_assert_required, _assert_string],
             "key": [_assert_required, _assert_string],
-            "value": [_assert_required, _assert_string],
+            "value": [_assert_string],
```

Comment:
Shall we change `_assert_required` to be 
```
def _assert_required(x):
    assert x is not None
```
and regard empty string as a normal value.

Reply:
See https://github.com/mlflow/mlflow/blob/41fa117beb90ead350dd8c0e367fb3a85249bc8d/mlflow/server/handlers.py#L298-L300